Assets/DecayZoneController.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DecayZoneController : MonoBehaviour
+{
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.gameObject.tag == "Player")
```

Comment:
Should this reference AI as well? so that they can hypothetically win the race

Reply:
lolololol yeah probably